grass7/raster/r.width.funct/r.width.funct.py

```diff
@@ -43,7 +43,7 @@
 import sys
 import os
 import matplotlib #required by windows
-matplotlib.use('wx') #required by windows
+matplotlib.use('wxagg') #required by windows
```

Comment:
```suggestion
matplotlib.use('WXAgg') #required by windows
```
(as e.g. here: https://github.com/OSGeo/grass/blob/master/gui/wxpython/iscatt/plots.py#L33)